cinema/serializers.py

```diff
@@ -68,13 +80,60 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSeatRowSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat")
+        read_only_fields = fields
```

Comment:
The `read_only_fields` attribute is incorrectly set to `fields`. It should be a tuple or list of field names that are read-only, such as `read_only_fields = ('row', 'seat')`.